PortfolioApp/Album/Views/AlbumTableViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+import RswiftResources
+
+final class AlbumTableViewController: UITableViewController {
+    private var modelAlbums: [AlbumData] = []
+    var albumsViewModel: AlbumsViewModel
+    
+    init(viewModel: AlbumsViewModel) {
+        self.albumsViewModel = viewModel
+        super.init(style: .plain)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        bindToAlbumsViewModel()
+    }
+    
+    private func bindToAlbumsViewModel() {
+        albumsViewModel.isLoadingData = { [weak self] isLoading in
+            if isLoading {
+                self?.tableView.refreshControl?.beginRefreshing()
+            } else {
+                self?.tableView.refreshControl?.endRefreshing()
+            }
+        }
+        
+        albumsViewModel.albumsViewData = { [weak self] albums in
+            self?.modelAlbums = albums
+            self?.tableView.reloadData()
+        }
+        
+        albumsViewModel.onLoad()
```

Comment:
This in not a binding, this is a call of function. It should be in the viewDidLoad